woocommerce/woocommerce
# e2e-builds: Private package for build logic of e2e

### All Submissions:






### Changes proposed in this Pull Request:

e2e packages used logic found in `packages/js/bin` to create builds. In an effort to isolate packages before converting the monorepo to Turborepo, that logic should be properly brought in as a dependency rather than referencing a bin folder that doesn't really belong in `packages/js`.

This PR creates a private package and declares a proper dependency in the consuming packages' `package.json`.

Partially fixes https://github.com/woocommerce/woocommerce/issues/32654

### How to test the changes in this Pull Request:

1. `pnpm install`
2. View the logs and confirm that 3 `e2e-*` packages are built without error.

### Other information:




### FOR PR REVIEWER ONLY:
